proto/kvrpcpb.proto

```diff
@@ -191,6 +191,7 @@ message BatchGetResponse {
 message ScanLockRequest {
     Context context = 1;
     uint64 max_version = 2;
+    uint64 limit = 3;
```

Comment:
if we add limit, do we need to add the start key?

Reply:
It is not necessary, because we get some locks in one region first and do resolve locks immediately. I think it's better than get all locks.

Reply:
I think it is easy to add when needed.

Reply:
Since we will enlarge the region size step by step, we may not able to scan the whole region once in the future. I prefer to add a start key.

Reply:
I think only `limit` is enough for this case and it works in the better way.
When there are too many locks in one region, the gc worker get some locks(not all) once, I think it is better to cleanup these locks at once, not to get the next batch of locks, because
1. Though we get 1000 locks per request, we may cleanup maybe 2000 locks once because lock is cleanup together if they are belong to the same transaction, so we can scan less locks in this region next time.
2. if there are too many locks in the region, I think it may be dangerous to get all locks and cleanup them at once. We may hold too many locks in gc worker(400MB in my case, and many times larger if gc works parallel).

Reply:
If you delete these locks, the next time you may skip these tombstone keys at first

Reply:
> I think it is better to cleanup these locks at once install of getting the next batch of locks
> I think it may be dangerous to get all locks and cleanup them at once

sorry, I don't know what do you really want to do, cleanup at once or not?

Reply:
yes, you are right, I want to cleanup them at once, so in next scan, we won't see the locks, they have been cleanup totally, that's why we don't need `start key`.

Reply:
here it is the RocksDB tombstone key

Reply:
If you delete the keys, the keys become tombstone keys in RocksDB, so if you still scan the locks with the region start key, we will skip the tombstone keys in RocksDB at first.

Reply:
Got it, if so, I think `start key` is better.